src/Symfony/Component/Validator/Constraints/NotCompromisedPasswordValidator.php

```diff
@@ -91,6 +91,10 @@ public function validate($value, Constraint $constraint)
         }
 
         foreach (explode("\r\n", $result) as $line) {
+            if (false === strpos($line, ':')) {
```

Comment:
```suggestion
            if (!str_contains($line, ':')) {
```

Reply:
Doesn't this break compatibility with PHP < 8 as `str_contains` is only available in PHP >= 8?

Reply:
The validator component for 5.4 has a dependency on polyfill-php80, so this is usable.